qcodes/interactive_widget.py

```diff
@@ -532,3 +532,6 @@ def experiments_widget(
     tab = create_tab(do_display=False)
     grid = _experiment_widget(data_sets, tab)
     return VBox([title, tab, grid])
+
+
+__all__ = ["experiments_widget"]
```

Comment:
let's add it since it's neat? :)
```suggestion
__all__ = ["experiments_widget", "nested_dict_browser"]
```